pkg/gatherers/clusterconfig/olm_operators.go

```diff
@@ -36,17 +27,32 @@ type csvRef struct {
 	Version   string
 }
 
-// GatherOLMOperators collects list of installed OLM operators.
-// Each OLM operator (in the list) contains following data:
+// GatherOLMOperators Collects the list of installed OLM operators. Each OLM operator (in the list) contains
+// following data:
 // - OLM operator name
 // - OLM operator version
 // - related ClusterServiceVersion conditions
 //
-// * See: docs/insights-archive-sample/config/olm_operators
-// * Location of in archive: config/olm_operators
-// * Id in config: clusterconfig/olm_operators
-// * Since versions:
-//   - 4.7+
+// ### API Reference
+// None
+//
+// ### Sample data
+// - docs/insights-archive-sample/config/olm_operators
+//
+// ### Location in archive
+// - `config/olm_operators`
+//
+// ### Config ID
+// `clusterconfig/olm_operators`
+//
+// ### Released version
+// - 4.7.0
+//
+// ### Backported versions
+// - 4.6.26+
```

Comment:
It seems this hasn't been backported.

Reply:
I have a different information based on [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1935574) and [RN](https://access.redhat.com/errata/RHBA-2021:1232)

Reply:
I was probably wrong here. You can ignore.